pkg/commands/init.go

```diff
@@ -30,6 +34,26 @@ type appInitOptions struct {
 	workloadType string
 }
 
+type CompStatus string
+
+const (
+	compStatusInitializing    CompStatus = "initializing"
+	compStatusInitFail        CompStatus = "initfail"
+	compStatusInitialized     CompStatus = "initialized"
+	compStatusDeploying       CompStatus = "deploying"
+	compStatusDeployFail      CompStatus = "deployfail"
+	compStatusDeployed        CompStatus = "deployed"
+	compStatusHealthChecking  CompStatus = "healthchecking"
```

Comment:
Will `healthChecking` be more proper?

Reply:
It seems string doesn't make sense here, I will change them into iota enum..